common/lib/dependabot/shared_helpers.rb

```diff
@@ -87,7 +87,7 @@ def self.run_helper_subprocess(command:, function:, args:, env: nil,
       if ENV["DEBUG_FUNCTION"] == function
         puts helper_subprocess_bash_command(stdin_data: stdin_data, command: cmd, env: env)
         # Pause execution so we can run helpers inside the temporary directory
-        byebug # rubocop:disable Lint/Debugger
+        binding.break # rubocop:disable Lint/Debugger
```

Comment:
Could have used the generic `debugger` here which is aliased in most (all?) debuggers, but 🤷

Reply:
Today I learned!